src/pymodaq_plugins_keithley/__init__.py

```diff
@@ -1,8 +1,18 @@
+import os
 from pathlib import Path
-from pymodaq.utils.logger import set_logger  # to be imported by other modules.
+from pymodaq.utils.logger import set_logger
+import pymodaq_plugins_keithley as plugin
+import pymodaq_plugins_keithley.utils as utils
+logger = set_logger('plugin.__init__', add_to_console=False)
 
-from .utils import Config
-config = Config()
+from pymodaq_plugins_keithley.utils import Config_keithley
+config = utils.Config()
+config_keithley = utils.Config_keithley()
+
+resources_path = plugin.__path__[0]+"/resources"
+toml_modules = [f for f in os.listdir(resources_path) if "module" in f and ".toml" in f]
+for file in toml_modules:
+    exec("config_k" + str(file[-9:-5]) + " = " + "utils.Config_k" + str(file[-9:-5]) + "()")
```

Comment:
what is that for? sounds like prone to bugs or even execution of harmful code

Reply:
It was generating dynamic classes from each .toml. But indeed as we talked about it during the ANF and after discussing with Loic about the possible bugs, I totally reworked the configuration approach to fit your config_template.toml.